pkg/binary/envoy/start_test.go

```diff
@@ -0,0 +1,162 @@
+// Copyright 2019 Tetrate
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package envoy
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/tetratelabs/getenvoy/pkg/binary"
+)
+
+const debugDir = "~/.getenvoy/debug/1"
```

Comment:
Can we use the temp dir Setup/TearDown in TestMain? 😄

Reply:
this is a proper unit test, so it doesn't actually write anything.

Reply:
I'll rename to fakeDebugDir to clarify

Reply:
sg!